src/renderer/features/assets/AssetsPortfolio/model/portfolio-model.ts

```diff
@@ -0,0 +1,174 @@
+import { createEffect, createEvent, createStore, sample } from 'effector';
+import { once } from 'patronum';
+
+import { Account, AccountId, Balance, Chain, ChainId, TokenAsset, TokenBalance, Wallet } from '@shared/core';
+import { networkModel, networkUtils } from '@entities/network';
+import { accountUtils, walletModel, walletUtils } from '@entities/wallet';
+import { AssetsListView } from '@entities/asset';
+import { balanceModel, balanceUtils } from '@entities/balance';
+import { sumTokenBalances, tokensService } from '../lib/tokensService';
+
+const setActiveView = createEvent<AssetsListView>();
```

Comment:
I personally don't mind, but I thought we usually name the event in the past form for events. It would be `activeViewSet` and `accountsSet` in that case.

Reply:
good point